### DISCUSSION TITLE
Adding borders to colorbar

### DISCUSSION THREAD

**Discussion by AlFontal** [has2k1/plotnine#652] [Q&A]
I've been trying to add borders to colorbars to be consistent with other colorbars generated in `matplotlib`, which by default contain black borders.

I could always remove the borders on the `matplotlib` side, but I've come to quite like the aesthetics of a bordered colorbar, and I was wondering whether it's even possible to replicate that.

As an extra, `ggplot2`'s current release allows to do this by using the `frame.colour` attribute in `guide_colorbar`.

Does anyone know whether there is any way of doing this through the current `plotnine` API?

Minimal example of what I mean:

```python
df = pd.DataFrame({
     'value': np.random.randn(100),
     'variable': np.tile(['A', 'B', 'C', 'D', 'E'], 20),
     'time': np.repeat(np.arange(1, 21), 5)
})
```

```python
# in matplotlib
plt.imshow(df.pivot('time', 'variable', 'value'))
plt.colorbar()
```

<img width="300" alt="imagen" src="https://user-images.githubusercontent.com/10958332/207078375-24181036-0e73-4dca-a0cc-5a3e9ff53380.png">

```R
# ggplot2 in R
ggplot(df) +
    aes(variable, time, fill = value) +
    geom_tile() +
    scale_fill_continuous(type = 'viridis',
                          guide = guide_colorbar(frame.colour = 'black'))
```
![imagen](https://user-images.githubusercontent.com/10958332/207080391-01ba89e3-78ef-4d19-b02a-8e6cddbdac03.png)

**Comment by has2k1** [comment]
It is not possible. Create an issue for a feature request.

**Reply by AlFontal** [reply]
Got it, thanks! Just did it.
